generator/generator.yml

```diff
@@ -361,3 +361,19 @@ modules:
       - picoMobileMIB
       - picoIPv4MIB
       - picoIPv6MIB
+
+  raritan:
+    walk:
+      - sysUpTime
+      - interfaces
```

Comment:
I'm thinking maybe we should leave `interfaces` off this module. People can get this from `if_mib`.

Reply:
Done  :)